src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html

```diff
@@ -3,3 +3,20 @@
         ${contribSelectFragmentsHtml}
     </select>
 </div>
+<div>
+    <a href='#' data-toggle="modal" data-target="#moreInfoEqualShareModal">${equalShareHelp}</a>
+</div>
+<div id='moreInfoEqualShareModal' class="modal fade" role="dialog">
```

Comment:
Any `id` or `class` should use lowercase words separated by hyphens. While there are currently many violations of this, it's still better to follow the conventions.